frontend/src/components/AdminDash/Filters.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Button, Form, FormGroup, Modal, ModalHeader, ModalBody, ModalFooter, CustomInput, Label } from 'reactstrap';
+
+class Filters extends React.Component {
+  render() {
+    const { show, toggleCallback } = this.props;
+    return (
+      <Modal isOpen={show}>
+        <ModalHeader toggle={toggleCallback}>Filters</ModalHeader>
+        <ModalBody>
+          <FormGroup>
```

Comment:
Do the inputs on this form get submitted anywhere rn? If so may eventually wanna check out the `formik` components I made and see if they might be useful here

Reply:
They do not. this was Kanav's modal using the reactstrap template. Will need to use `formik`